util/web3Storage.js

```diff
@@ -0,0 +1,25 @@
+import { Web3Storage, File } from 'web3.storage'
+
+export function getAccessToken() {
+  return process.env.WEB3STORAGE_TOKEN
```

Comment:
Are we planning to let each community choose their data storage and add keys, or should we just use a single web3storage account sponsored by IndieDAO for now?

Reply:
I think we can use a single account sponsored by us for now since the data is not that big. Later we can think in a more scalable solution with separate accounts.